src/main/scala/com/ing/wbaa/airlock/sts/service/db/dao/STSUserDAO.scala

```diff
@@ -19,6 +20,7 @@ trait STSUserDAO extends LazyLogging {
 
   private[this] val MYSQL_DUPLICATE__KEY_ERROR_CODE = 1062
   private[this] val USER_TABLE = "users"
+  private[this] val USER_GROUP_TABLE = "user_groups"
```

Comment:
I would suggest considering moving groups table with its methods to separate DAO trait like we have for user and token. So really mapping per table. I see some reason for this is combined query, but maybe groupsDao can be mixed in in userDao?

Reply:
`def insertUserGroups` can be moved to a separate class but to not create two sql queries the `def getUserSecretKeyAndIsNPA` uses the ***user_groups*** table so I changed the DAO name.